observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,31 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6-1_{context}"]
+== Network Observability Operator 1.6.1
+The following advisory is available for the Network Observability Operator 1.6.1:
+
+* link:https://access.redhat.com/errata/RHSA-2024:4785[2024:4785 Network Observability Operator 1.6.1]
+
+[id="network-observability-operator-CVE-1-6-1_{context}"]
+=== CVEs
+* link:https://access.redhat.com/errata/RHSA-2024:4237[RHSA-2024:4237]
+* link:https://access.redhat.com/errata/RHSA-2024:4212[RHSA-2024:4212]
+
+[id="network-observability-operator-1.6.1-bug-fixes_{context}"]
+=== Bug fixes
+* Previously, packet drops information, such as cause and TCP state, was only available in the Loki datastore and not in Prometheus. For that reason, the drop statistics in the OpenShift web console plugin *Overview* was only available with Loki. Now, packet drops information is also added to metrics, so you can view drops statistics when Loki is disabled. (link:https://issues.redhat.com/browse/NETOBSERV-1649[*NETOBSERV-1649*])
+* When the eBPF agent `PacketDrop` feature is enabled, and sampling is configured to a value greater than `1`, reported dropped bytes and dropped packets ignore the sampling configuration. While this is done on purpose to not miss any drops, a side effect is that the reported proportion of drops versus non-drops becomes biased. For example, at a very high sampling rate, such as `1:1000`, it is likely that almost all the traffic appears to be dropped when observed from the console plugin. (link:https://issues.redhat.com/browse/NETOBSERV-1676[*NETOBSERV-1676*])
```

Comment:
The jira RN text is still for to the 1.6.0 known issue, it should be updated such as:

```suggestion
* When the eBPF agent `PacketDrop` feature is enabled, and sampling is configured to a value greater than `1`, reported dropped bytes and dropped packets was ignoring the sampling configuration. While this was done on purpose to not miss any drops, a side effect was that the reported proportion of drops versus non-drops became biased. For example, at a very high sampling rate, such as `1:1000`, it was likely that almost all the traffic appears to be dropped when observed from the console plugin. The new behaviour is to honour the sampling configuration with dropped bytes and packets. (link:https://issues.redhat.com/browse/NETOBSERV-1676[*NETOBSERV-1676*])
```

Reply:
(I also updated jira RN)

Reply:
Ok thank you. Addressed.